### ISSUE TITLE
MAC BOINC libraries fail to build

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#807]
**Reported by tstrunk on 2 Nov 39057563 11:33 UTC**
buildcurl.sh was not completely updated to c-ares 1.60 and doesn't build automatically anymore.[../c-ares-1.5.3
sudo make install
```
should point to c-ares-1.6.0, also sudo is not needed and should therefore be avoided here. [[br]([br]]
buildcurl.sh:88-89
```
cd)]
Same in buildcurl.sh:202
```
sudo rm -Rf /tmp/installed-c-ares/
```

**Comment by romw** [comment]
**Commented by charlief on 31 Mar 39059067 06:40 UTC**
Replying to [tstrunk](ticket:830):
> buildcurl.sh was not completely updated to c-ares 1.60 and doesn't build automatically anymore.[buildcurl.sh:88-89
> ```
> cd ../c-ares-1.5.3
> sudo make install
> ```
> should point to c-ares-1.6.0, also sudo is not needed and should therefore be avoided here. [[br]([br]]
>)]
> Same in buildcurl.sh:202
> ```
> sudo rm -Rf /tmp/installed-c-ares/
> ```

Thank you for bringing this to my attention.  Fixed for 6.6.2.

**Comment by romw** [comment]
**Commented by charlief on 17 Apr 39059084 03:06 UTC**
(In [16964]) Mac client: fix bug in build script so that curl-7.19.2 actually does build with c-ares 1.6.0.  Fixes #830.

**Comment by romw** [comment]
**Commented by charlief on 4 Nov 39059087 10:13 UTC**
(In [16965]) Mac client: fix bug in build script so that curl-7.19.2 actually does build with c-ares 1.6.0.  Fixes #830.

**Comment by romw** [comment]
**Commented by charlief on 14 Feb 39059090 18:13 UTC**
(In [16966]) Mac client: fix bug in build script so that curl-7.19.2 actually does build with c-ares 1.6.0.  Fixes #830.
